test/filterTicketsbyKeyword.test.ts

```diff
@@ -0,0 +1,415 @@
+import getNewTickets from "../src/services/zendesk/getNewTicketsService";
+import makeZendeskRequest from "../src/services/zendesk/authenticationService";
+import filterTicketsByKeyword from "../src/services/zendesk/filterTicketsByKeyword";
+import filterTicketsByKeyword2 from "../src/services/zendesk/filterTicketsByKeyword2";
+
+jest.mock('../src/services/zendesk/getNewTicketsService');
+const ticketsArray = [
+    {
+      url: 'https://grzegorzabchelp.zendesk.com/api/v2/tickets/4.json',
+      id: 4,
+      external_id: null,
+      created_at: '2022-05-26T07:58:12Z',
+      updated_at: '2022-05-26T07:58:12Z',
+      type: null,
+      subject: 'access',
+      raw_subject: 'access',
+      description: 'access',
+      priority: null,
+      status: 'open',
+      recipient: null,
+      requester_id: 393092207478,
+      submitter_id: 393092207478,
+      assignee_id: 393092207478,
+      organization_id: 5295847522845,
+      group_id: 5295868098205,
+      collaborator_ids: [],
+      follower_ids: [],
+      email_cc_ids: [],
+      forum_topic_id: null,
+      problem_id: null,
+      has_incidents: false,
+      is_public: true,
+      due_at: null,
+      tags: [],
+      custom_fields: [],
+      satisfaction_rating: null,
+      sharing_agreement_ids: [],
+      followup_ids: [],
+      ticket_form_id: 5295891567005,
+      brand_id: 5295847519133,
+      allow_channelback: false,
+      allow_attachments: true
```

Comment:
majority of the ticket fields here are obsolete - please remove unused ones and leave just the fields relevant for testing

Reply:
corrected